packages/renderer/src/lib/feedback/SendFeedback.svelte

```diff
@@ -44,9 +69,10 @@ function hideModal(): void {
     </div>
 
     {#if category === 'developers'}
-      <DevelopersFeedback onCloseForm={hideModal}/>
+      <DevelopersFeedback onCloseForm={hideModal} on:propsAreEmpty={handleUpdate}/>
```

Comment:
I would no use the `props` wording, might just provide a `onchange` method, so we keep with _traditional_ props, we can just ignore the content, just capture the event, and set the `allEmptyOrZero` to true or false